### PR TITLE
R30 Release Announcement

### PR DESCRIPTION
**Pull request by manojleaton**
Fixes #BLUI-5150 .

#### Changes proposed in this Pull Request:

- Added R30 Release Notes 

#### Screenshots / Screen Recording (if applicable)
Screenshot 2024-01-18 at 5 47 26 PM

#### To Test:

- yarn start

#### Any specific feedback you are looking for?

-

### CODE REVIEW

**Review by ektaghag-eaton — COMMENTED** [review]
Please update R30 Roadmap and And add new R31 roadmap details.
Screenshot 2024-01-18 at 9 01 06 PM

**Review by huayunh — COMMENTED** [review]
* I wonder if this should be BLUI 4.0. 
* The phrase "Material Design 3" is repeated three times in the summary. Consider making it more succinct.
* Avoid saying `@brightlayer-ui/colors/branding package`, as this is confusing. 
* Your "package summary" section is not used correctly. We divided it by frameworks so that React devs will read only react-related stuff. Some shared things, like templates, colors, and icons, are grouped together. Though it would be beneficial to call out the two deprecated packages.
* For the "5 new components have been added" line, the components need to be linked to dev doc.
* I am personally not a fan of "as per". I think there's only "per" or "according to" in modern American English.
* Changelogs need to link to the master branch.

File: src/docs/release-notes/R30/FullNotes.mdx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+## React Native Component Library
+
+-   Updated Component Library with several changes by utilizing React Native Paper V5 with Material Design V3.
+    -   Typography and IconWrapper have been removed.
+    -   5 new components have been added, namely [Chip](https://github.com/etn-ccis/blui-react-native-component-library/blob/dev/docs/Chip.md), [Grade](https://github.com/etn-ccis/blui-react-native-component-library/blob/dev/docs/Grade.md), [Icon](https://github.com/etn-ccis/blui-react-native-component-library/blob/dev/docs/Icons.md#icon-as-a-component), [IconSwitch](https://github.com/etn-ccis/blui-react-native-component-library/blob/dev/docs/IconSwitch.md) and [Overline](https://github.com/etn-ccis/blui-react-native-component-library/blob/dev/docs/Overline.md).
```
Comment (JeffGreiner-eaton) [inline src/docs/release-notes/R30/FullNotes.mdx:7]: you can link to master branch for all the components here

Comment (JeffGreiner-eaton) [reply src/docs/release-notes/R30/FullNotes.mdx:7]: @huayunh already mentioned this so ignore my comment but link out to component docs master...
https://brightlayer-ui-components.github.io/react-native/?path=/info/components-documentation--chip

File: src/docs/release-notes/R30/FullNotes.mdx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+## React Native Themes
+
+-   Updated themes to use Material Design 3 styled palettes and React Native Paper v5 theme structure.
+-   Added required custom color variants along with React Native Paper v5 default color variants.
+
+## Colors
+
+-   Moved Branding colors under @brightlayer-ui/colors package.
+-   Updated UI colors and Branding colors according to new Material Design 3.
+-   Removed dependency of @brightlayer-ui/types and moved type in @brightlayer-ui/colors package.
```
Comment (huayunh) [inline src/docs/release-notes/R30/FullNotes.mdx:19]:
```suggestion
-   Updated UI colors to match Material Design 3's color palette structure.
-   Moved branding colors under @brightlayer-ui/colors.
-   Removed dependency on @brightlayer-ui/types.
```

**Pull request merged**